worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageDriverImpl.java

```diff
@@ -114,9 +114,9 @@ public CoverageDriverImpl(CoverageSearchRepository coverageSearchRepository,
      * @return the current meaningful coverage update configuration
      */
     private CoverageUpdateConfig retrieveConfig() {
-        String updateMonths = propertiesApiService.getProperty(COVERAGE_SEARCH_UPDATE_MONTHS);
-        String stuckHours = propertiesApiService.getProperty(COVERAGE_SEARCH_STUCK_HOURS);
-        String override = propertiesApiService.getProperty(COVERAGE_SEARCH_OVERRIDE);
+        String updateMonths = propertiesService.getProperty(COVERAGE_SEARCH_UPDATE_MONTHS, "3");
+        String stuckHours = propertiesService.getProperty(COVERAGE_SEARCH_STUCK_HOURS, "24");
```

Comment:
Should these be hard-coded, or variables.

Reply:
They're just defaults in case the service isn't available. Will remove when there is more confidence.